### PR TITLE
feat: formatter case YYYY-MM-DD for our new datetime formatter

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Part of #2099

As described in the issue, we need that formatter case. This PR adds it with its accompanying tests.

### CODE REVIEW

File: src/utils/datetime/formatters.test.ts [hunk 1]
```diff
@@ -64,6 +65,12 @@ describe('the DateTime formatter', () => {
   }
 
   describe('formatting DateTimes in UTC', () => {
+    it('formats DateTimes in the default time YYYY-MM-DD in UTC', () => {
```
Comment (hoorayimhelping) [inline src/utils/datetime/formatters.test.ts:68]:
```suggestion
    it('formats DateTimes in the format YYYY-MM-DD in UTC', () => {
```
also I think for organization sake, keep the default case at the top of the file

**Review by hoorayimhelping — APPROVED** [review]
quick turnaround

File: src/utils/datetime/formatters.test.ts (hunk 1 above)
Comment (ChitlangeSahas) [reply src/utils/datetime/formatters.test.ts]: For sure! Done 👍

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 53 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2103)

**Pull request merged**